src/config/oidc-provider-configuration.ts

```diff
@@ -1,11 +1,18 @@
 import { Request } from "express";
-import { Configuration } from "oidc-provider";
+import { Configuration, type UnknownObject } from "oidc-provider";
 import { destroyAuthenticatedSession } from "../managers/session/authenticated";
 import epochTime from "../services/epoch-time";
 import { findAccount } from "../services/oidc-account-adapter";
 import policy from "../services/oidc-policy";
 import { renderWithEjsLayout } from "../services/renderer";
 
+//
+
+export interface OIDCContextParams extends UnknownObject {
```

Comment:
put this in a seperate file "types/oidc-conetext-params.ts" ?

add a com :

// this override the KoaContextWithOIDC["oidc"]["params"] type which is incomplete in the original lib